src/library/Uno.Material/Styles/Controls/v2/DatePicker.xaml

```diff
@@ -1,57 +1,177 @@
-                    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-                    xmlns:android="http://uno.ui/android"
-                    xmlns:ios="http://uno.ui/ios"
-                    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-                    xmlns:not_win="http://uno.ui/not_win"
-                    xmlns:toolkit="using:Uno.UI.Toolkit"
-                    xmlns:wasm="http://uno.ui/wasm"
-                    mc:Ignorable="ios android wasm not_win">
-
-	<x:Double x:Key="MaterialDateTimeFlyoutBorderThickness">1</x:Double>
-	<x:Double x:Key="MaterialDatePickerSpacerThemeWidth">1</x:Double>
-	<x:Double x:Key="MaterialDatePickerHeight">53</x:Double>
-	<StaticResource x:Key="MaterialDatePickerFlyoutPresenterBackgroundBrush"
-	                ResourceKey="SurfaceBrush" />
-	<Thickness x:Key="MaterialDatePickerHostPadding">24,24,8,8</Thickness>
-
-	<StaticResource x:Key="MaterialDatePickerFlyoutPresenterBorderBrush"
-	                ResourceKey="OnSurfaceFocusedBrush" />
-	<SolidColorBrush x:Key="MaterialDatePickerFlyoutPresenterHighlightFill"
-	                 Opacity="0.20"
-	                 Color="{ThemeResource PrimaryColor}" />
-	<x:Double x:Key="MaterialDatePickerFlyoutElevation">8</x:Double>
-	<StaticResource x:Key="MaterialDatePickerFlyoutPresenterSpacerFill"
-	                ResourceKey="OnSurfaceFocusedBrush" />
-
-	<SolidColorBrush x:Key="MaterialDatePickerBackgroundColorBrush"
-	                 Opacity="0.12"
-	                 Color="{ThemeResource OnSurfaceColor}" />
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:android="http://uno.ui/android"
+					xmlns:ios="http://uno.ui/ios"
+					xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+					xmlns:not_win="http://uno.ui/not_win"
+					xmlns:toolkit="using:Uno.UI.Toolkit"
+					xmlns:wasm="http://uno.ui/wasm"
+					mc:Ignorable="ios android wasm not_win">
+
+	<x:Double x:Key="DateTimeFlyoutBorderThickness">1</x:Double>
+	<x:Double x:Key="DatePickerSpacerThemeWidth">1</x:Double>
+	<x:Double x:Key="DatePickerHeight">53</x:Double>
+	<x:Double x:Key="DatePickerFlyoutElevation">8</x:Double>
+	<x:Double x:Key="DatePickerFlyoutButtonOpacityPressed">0.65</x:Double>
+	<x:Double x:Key="DatePickerFlyoutButtonOpacityDisabled">0.65</x:Double>
+	<x:Double x:Key="DatePickerFlyoutPresenterWidth">296</x:Double>
+	<x:Double x:Key="DatePickerFlyoutPresenterMinWidth">296</x:Double>
+	<x:Double x:Key="DatePickerFlyoutPresenterMaxHeight">398</x:Double>
+	<x:Double x:Key="DatePickerFlyoutPresenterAcceptDismissHostGridHeight">41</x:Double>
+	<x:Double x:Key="DatePickerFlyoutPresenterHighlightHeight">40</x:Double>
+	<Thickness x:Key="DatePickerHostPadding">24,24,8,8</Thickness>
+	<Thickness x:Key="DatePickerFlyoutButtonPadding">0</Thickness>
+	<CornerRadius x:Key="DatePickerCornerRadius">4,4,0,0</CornerRadius>
+
+	<ResourceDictionary.ThemeDictionaries>
```

Comment:
Looks good! Only thing is the formatting, if you run Xaml Styler it should fix this, unless it's overridden in your XamlStyler settings but the x:Key and the ResourceKey should be on the same line for the StaticResource. And we shouldn't need a blank line between each StaticResource declaration unless its between groupings of the resources by property

So it should look organized by property like we do in other styles like [here](https://github.com/unoplatform/Uno.Themes/blob/d5d803370cc99e11e3e895992a220b95abec477a/src/library/Uno.Material/Styles/Controls/v2/TextBox.xaml#L27-L50)